lib/validators/calculating_augmented_records.rb

```diff
@@ -20,7 +19,7 @@ def parse_and_save_record(record)
     # create a temporary saved record copy to do calculations on, then delete it
```

Comment:
Should change this comment, now that the record is no longer saved.

Reply:
Looking back at this we should maybe make a few more considerations here, `parse_record` passes over to CqmExecutionCalc which saves IndividualResults to the DB and also returns the individual results. Do we always need to save IndividualResults to the DB/do we ever need IndividualResults saved to the DB?